### PR TITLE
*: merge release-1.9.7 to master

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

**Comment by tariq1890** [comment]
@lilic requesting your review on this

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-## MASTER
```
Comment (lilic) [inline CHANGELOG.md:1]: We want to keep this no?

Comment (tariq1890) [reply CHANGELOG.md:1]: I don't think so. We've never really recorded `master`(`main`) branch changelogs. We could start that practice in a new PR IMO

**Pull request merged**